.cirrus/schema_test.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+set -euo pipefail
+
+cd "$(dirname "$0")/../tests/dynamic-compatibility-test"
+
+go mod tidy
+go test -timeout=0 -v schema_test.go
```

Comment:
It's super cool that you added more tests! It will be then easier to spot the existence of drift from our desired tagging logic! However, I consider this as static tests, as they do not require to apply our helm chart to a cluster and run SonarQube. Can we move schema_test.go to the `unit-compatibility-test` folder?

Reply:
Partially agree.
I think that the `unit-compatibility-test` tests are dependant on Kubernetes versions, while these new tests aren't/shouldn't.

Not only that, but the fixtures that we use for other tests are stored within the `unit-compatibility-test`, therefore putting these new tests there could lead to more confusion on the long run.